src/Orckestra.Composer.Cart/Services/Order/OrderHistoryViewService.cs

```diff
@@ -569,6 +570,17 @@ public virtual async Task<CompleteCheckoutViewModel> SaveEditedOrderAsync(string
                 throw new InvalidOperationException($"Cannot save edited order #${orderNumber} as it doesn't exist.");
             }
 
+            var removeInvalidLineItemsParam = new RemoveInvalidLineItemsParam
+            {
+                CartName = order.Id.Replace("-", "").ToLower(),
+                CartType = CartConfiguration.OrderDraftCartType,
```

Comment:
try order.Id.ToString("N")

Reply:
Id is a string there, I think that will not work